docs/src/remap/remap.adoc

```diff
@@ -2216,10 +2199,10 @@ To see fields in the global emcStatus structure, try this:
 
 You need to have LinuxCNC started from a terminal window to see the results.
 
-=== How Tx (Prepare Tool) works
+=== How `T`__x__ (Prepare Tool) works
 
 [[remap:interpreter-action-on-t]]
-.Interpreter action on a Tx command
+.Interpreter action on a `T`__x__ command
```

Comment:
IMHO monospace font in headings and sub-headings looks ugly. Anyone who agree/disagree?

![grafik](https://user-images.githubusercontent.com/67957916/230445645-2949ab47-a7f0-4b24-b5c4-05e4d5616f89.png)

Reply:
I find it looks very nice.